glide.lock

```diff
@@ -13,7 +17,7 @@ imports:
 - name: github.com/fsnotify/fsnotify
   version: 629574ca2a5df945712d3079857300b5e4da0236
 - name: github.com/gogo/protobuf
-  version: ba06b47c162d49f2af050fb4c75bcbc86a159d5c
```

Comment:
are all the version upgrade for the libraries needed for this change?

Reply:
Those version changes in this file are auto-added when adding/updating the lib in glide.yaml.